scarlet/source.py

```diff
@@ -36,17 +36,6 @@ def get_pixel_sed(sky_coord, observation):
 
     pixel = observation.frame.get_pixel(sky_coord)
     sed = observation.images[:, pixel[0], pixel[1]].copy()
-    if observation.frame.psfs is not None:
-        # Account for the PSF in the intensity
-        sed /= observation.frame.psfs.max(axis=(1, 2))
```

Comment:
Because it does something that the user doesn't expect. It gives you the SED of a pixel *and* makes the first step of a two-step correction for PSF width changes. That is highly confusing. It's conceptually better to keep the SED rescaling to the init methods.

Reply:
In that case it might still be better to do the rescaling in the function but introduce a `rescale_by_psf` parameter to optionally rescale by the PSF. That would help avoid duplicate code but still make it clear what is being done.

Reply:
I disagree. This entire operation only makes sense when you also do the multiplication with the blend frame PSF because it's in essence a matching operation. That's what I meant by two-step procedure. It's very unclear what this rescaling does if the two parts are performed in very different regions of the code.

Reply:
I get your point, but in that case we should probably have a function like `_get_normalized_sed` that implements lines 108-117 as a function that `init_extended_source` and `init_combined_extended_source` can both call.

Reply:
new function `get_psf_sed` that does that.